### PR TITLE
Reorder the entries in the aggregation file according to the given cm…

### PR DESCRIPTION
**Pull request by jpa198**
…t file order

### CODE REVIEW

File: srf_generation/source_parameter_generation/generate_realisations_from_gcmt.py [hunk 1]
```diff
@@ -314,6 +314,17 @@ def main():
     worker_pool = pool.Pool(processes=n_processes)
```
Comment (claudio525) [inline srf_generation/source_parameter_generation/generate_realisations_from_gcmt.py:314]: Gotta close those pools man

File: srf_generation/source_parameter_generation/generate_realisations_from_gcmt.py [hunk 2]
```diff
@@ -314,6 +314,17 @@ def main():
...
     worker_pool.starmap(generate_fault_realisations, messages)
 
+    if args.aggregate_file is not None:
+        ordered_rels = [
+            get_realisation_name(pid, i + 1) for pid in pids for i in faults[pid]
+        ]
+
+        agg = pd.read_csv(args.aggregate_file)
+        agg = agg.sort_values(
```
Comment (jasonmotha) [inline srf_generation/source_parameter_generation/generate_realisations_from_gcmt.py:323]: Consider using pandas categoricals to sort the df?

```df['LSE_cat'] = pd.Categorical(
    df['LSE'], 
    categories=['Oands','Wetnds','Develd','Cn','Soys','Otherg','Wht'], 
    ordered=True
)
df.sort_values('LSE_cat')```

Comment (jpa198) [reply srf_generation/source_parameter_generation/generate_realisations_from_gcmt.py:323]: I considered it, but that involves a lot more faffing around to achieve the same result

File: srf_generation/source_parameter_generation/generate_realisations_from_gcmt.py (hunk 1 above)
Comment (jpa198) [reply srf_generation/source_parameter_generation/generate_realisations_from_gcmt.py]: Pools closed

**Review by jasonmotha — APPROVED** [review]

**Review by claudio525 — APPROVED** [review]

**Pull request merged**